### PR TITLE
Add image fallback support + using prop-types

### PR DESCRIPTION
**Pull request by janoist1**

### CODE REVIEW

File: assets/js/src/components/ResponsiveImageSize.jsx [hunk 1]
```diff
@@ -116,6 +119,23 @@ class ResponsiveImageSize extends React.Component {
 
 
   /**
+   * Called when image loading failed
+   */
+  onError(e) {
+    if (this.props.fallbackImage) {
+      this.setState({
+        ...this.state,
```
Comment (mikefey) [inline assets/js/src/components/ResponsiveImageSize.jsx]: Does this do something different than just calling `this.setState({ imagePath: this.props.fallbackImage })` ?

**Review by mikefey — COMMENTED** [review]
Thanks for this pull request! Just had one small comment, but otherwise everything looks good.

File: assets/js/src/components/ResponsiveImageSize.jsx (hunk 1 above)
Comment (janoist1) [reply assets/js/src/components/ResponsiveImageSize.jsx]: Yes, you're right. It does a merge, no need for doing it there. Thanks!

**Comment by janoist1** [comment]
Thanks for the module that has saved some of my time. :)
I have updated that bit.

**Pull request merged**